packages/cli/src/lib/snapshot/snapshot.spec.ts

```diff
@@ -161,7 +161,8 @@ describe('Snapshot', () => {
 
     beforeEach(async () => {
       mockedDryRunReport = getSuccessDryRunReport(snapshotId);
-      [snapshot] = await getSnapshot(mockedDryRunReport);
+      const a = await getSnapshot({reports: [mockedDryRunReport]});
+      snapshot = a[0];
```

Comment:
```suggestion
      snasphot = (await getSnapshot({reports: [mockedDryRunReport]}))[0];
```
1. `a` is really not useful here
2. `a` is not a good var name

1+2=> rm -f